### ISSUE TITLE
Windows `use client` console log not working.

### ISSUE THREAD

**Issue by lvergergsk** [vercel/next.js#54568]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
On my windows machine:

Processor	Intel(R) Core(TM) i9-10850K CPU @ 3.60GHz   3.50 GHz
Ram		64.0 GB

Edition:	Windows 11 Pro
Version:	22H2
Release Date:	‎2023/‎02/‎09
OS Build:	22621.2134
```

### Link to the code that reproduces this issue or a replay of the bug

https://github.com/lvergergsk/playground-20230825-003

### To Reproduce

I've made an example repo: https://github.com/lvergergsk/playground-20230825-003

```
"use client"

export default function Home() {
  return (
    <main className="flex min-h-screen flex-col items-center justify-between p-24">
        <button onClick={console.log}>hi</button>
    </main>
  )
}
```

This simple code does not work on windows, but work on MacBook

### Describe the Bug

I can't see the console log on chrome console when I click `hi`, but all works fine when I run this code on macbook.
I only verified on one Windows machine because I only have one.

Fail on `13.4.20-canary.6`.
Fail on `13.4.19`
Fail on  `13.4.17`
Success on `13.4.16` <- This version is fine, even on my Windows machine.

Fail when using `npm run dev`
Fail when using `npm run build` and `npm run start`

(Sorry, I don't have time to contribute code, just report the observation.)

### Expected Behavior

When click `hi` on screen, it should log the event on chrome console.

### Which browser are you using? (if relevant)

Chrome 116.0.5845.111

### How are you deploying your application? (if relevant)

npm run dev

**Comment by balazsorban44** [comment]
Duplicate of https://github.com/vercel/next.js/issues/54269

TLDR; Upgrade Node.js
